crux_core/src/capability.rs

```diff
@@ -93,6 +364,28 @@ where
         CapabilityContext { inner }
     }
 
+    /// Transform the CapabilityContext into one maps each event passed to
+    /// `update_app` using the provided function.
```

Comment:
This paragraph definitely needs fixed (i'd make a suggestion but I'm not certain what it was meant to say)